src/smart-components/group/groups.js

```diff
@@ -62,9 +68,22 @@ const Groups = ({ fetchGroups, isLoading, pagination, history: { push }, groups
         variant="primary"
         aria-label="Create group"
       >
-        Add group
+        Create a group
       </Button>
-    </Link>
+    </Link>,
+    {
+      label: 'Edit group',
+      props: {
+        isDisabled: selectedRows.length === 1 ? false : true
+      },
+      onClick: () => push(`groups/edit/${selectedRows[0].uuid}`)
```

Comment:
I think you need the leading `/` in 
```jsx
push(`/groups/edit/${selectedRows[0].uuid}`)
```
You want to replace the whole path not append something to it.